plugins/pluginmanagers/lz-n.nix

```diff
@@ -191,9 +201,9 @@ nixvim.plugins.mkNeovimPlugin {
     };
 
   extraConfig = cfg: {
-    globals.lz_n = modules.mkAliasAndWrapDefsWithPriority id options.plugins.lz-n.settings;
+    globals.lz_n = lib.modules.mkAliasAndWrapDefsWithPriority id options.plugins.lz-n.settings;
```

Comment:
If you're doing a cleanup-commit (remove `with lib`) maybe we could have another cleanup commit that uses `extraConfig = cfg: opts: {` instead of using an explicit `options.plugins.lz-n`?

Reply:
Done.